### PR TITLE
test: add integration test for return document

### LINKED ISSUE #200: ReturnDocument tests

### PR DESCRIPTION
**Pull request by StarryFolf**

### CODE REVIEW

File: tests/Application.Tests.Integration/Borrows/Commands/ReturnDocumentTests.cs [hunk 1]
```diff
@@ -0,0 +1,152 @@
...
+        
+        // Act
+        var result = await SendAsync(command);
+        
+        // Assert
+        result.Id.Should().Be(borrow.Id);
+        result.Status.Should().Be(BorrowRequestStatus.Returned.ToString());
+        var documentResult = await FindAsync<Document>(result.DocumentId);
+        documentResult!.Status.Should().Be(DocumentStatus.Available);
+        result.BorrowTime.Should().NotBeAfter(borrow.BorrowTime.ToDateTimeUnspecified());
```
Comment (ChienNQuang) [inline tests/Application.Tests.Integration/Borrows/Commands/ReturnDocumentTests.cs]: I this necessary?

Comment (ChienNQuang) [reply tests/Application.Tests.Integration/Borrows/Commands/ReturnDocumentTests.cs]: Is*

**Review by ChienNQuang — COMMENTED** [review]
Just a clarification question

File: tests/Application.Tests.Integration/Borrows/Commands/ReturnDocumentTests.cs (hunk 1 above)
Comment (StarryFolf) [reply tests/Application.Tests.Integration/Borrows/Commands/ReturnDocumentTests.cs]: cường added this, not me

Comment (ChienNQuang) [reply tests/Application.Tests.Integration/Borrows/Commands/ReturnDocumentTests.cs]: @kaitoz11

Comment (kaitoz11) [reply tests/Application.Tests.Integration/Borrows/Commands/ReturnDocumentTests.cs]: Wtf! Just to be sure man! Who knows when shit gonna break?
Imagine a sunny-day, some sleepy guy refactor this mo'ah fakka and casually delete a line of code.
If ya don't want to do it this way then we need to pull out another test method for that specific check just to test whether someone shit on it or not....

Comment (kaitoz11) [reply tests/Application.Tests.Integration/Borrows/Commands/ReturnDocumentTests.cs]: Ehem... I sincerely sorry for my brainless actions....
I see my problem.

Comment (kaitoz11) [reply tests/Application.Tests.Integration/Borrows/Commands/ReturnDocumentTests.cs]: fixed at https://github.com/ProFile-org/ProFile-BE/pull/206/commits/b5ec2c8e2d5b1941a0b1777b2fa202f95acbef5d

**Pull request closed without merging**